.github/FUNDING.yml

```diff
@@ -0,0 +1,2 @@
+github: [f3l1x]
+custom: ["https://contributte.org/partners.html"]
```

Comment:
Sorry, I do not aggree with that. Pls have a look at at my comments here: https://github.com/contributte/datagrid-nette-database-data-source/pull/18#issuecomment-742332988
Thanks for understanding